.ruby-version

```diff
@@ -1 +1 @@
-2.6.3
+2.6
```

Comment:
setup-ruby support `2.4`, `2.5`, `2.6`, and `2.7`.
ref: https://github.com/actions/setup-ruby

(Until now the job failed.)